app/models/bank_statement_item.rb

```diff
@@ -71,7 +73,7 @@ class BankStatementItem < Ekylibre::Record::Base
     if (debit.nonzero? && credit.nonzero?) || (debit.zero? && credit.zero?)
       errors.add(:credit, :unvalid_amounts)
     end
-    if bank_statement && transfered_on
+    if bank_statement && transfered_on && started_on && stopped_on
```

Comment:
This doesn't do what we want.
Ex:
`started_on`: nil
`stopped_on`: 14/03/2019
`transfered_on`: 15/03/2019
`transfered_on` > `stopped_on` so we should have an error.

Reply:
Okay, started_on cannot be blank, alright.